docs/dev/getting-started/initial-setup/symfony-application/contao-5.3.md

```diff
@@ -0,0 +1,614 @@
+            login_throttling: ~
+
+            contao_login:
+                remember_me: true
+
+            remember_me:
+                secret: '%kernel.secret%'
+                remember_me_parameter: autologin
+                token_provider:
+                    doctrine: true
+
+            logout:
+                path: contao_frontend_logout
+
+    access_control:
+        - { path: ^%contao.backend.route_prefix%/login$, roles: PUBLIC_ACCESS }
+        - { path: ^%contao.backend.route_prefix%/logout$, roles: PUBLIC_ACCESS }
+        - { path: ^%contao.backend.route_prefix%(/|$), roles: ROLE_USER }
+        - { path: ^/, roles: [PUBLIC_ACCESS] }
+```
+
+Last part of this mandatory configuration is for the logging component. There is a development and a production configuration.
+The development configuration is located in `config/packages/monolog.yaml`.
+
+{{% notice tip %}}
+This is an example configuration and you can adjust this to your custom needs if required.
+{{% /notice %}}
+
+```yaml
+monolog:
+    handlers:
+        contao:
+            type: service
+            id: contao.monolog.handler
+
+        main:
+            type: rotating_file
+            max_files: 10
+            path: '%kernel.logs_dir%/%kernel.environment%.log'
+            level: debug
+            channels: ['!doctrine', '!event', '!php']
+
+        console:
+            type: console
+            bubble: false
+            verbosity_levels:
+                VERBOSITY_VERBOSE: INFO
+                VERBOSITY_VERY_VERBOSE: DEBUG
+            channels: ["!event", "!doctrine", "!console"]
+
+        console_very_verbose:
+            type: console
+            bubble: false
+            verbosity_levels:
+                VERBOSITY_VERBOSE: NOTICE
+                VERBOSITY_VERY_VERBOSE: NOTICE
+                VERBOSITY_DEBUG: DEBUG
+            channels: ["!event", "!doctrine", "!console"]
+
+when@prod:
+    monolog:
+        handlers:
+            contao:
+                type: service
+                id: contao.monolog.handler
+
+            main:
+                type: fingers_crossed
+                action_level: error
+                handler: nested
+                excluded_http_codes: [400, 401, 403, 404]
+
+            nested:
+                type: rotating_file
+                max_files: 10
+                path: '%kernel.logs_dir%/%kernel.environment%.log'
+                level: info
+
+            console:
+                type: console
+                process_psr_3_messages: false
+                channels: ["!event", "!doctrine"]
+```
+
+Now you're good to install the Contao Core bundle.
+
+## Install the Contao Core Bundle
+
+```
+$ composer require --with-all-dependencies \
+    contao/conflicts:@dev \
+    contao/core-bundle:"5.3.*" \
+    symfony/monolog-bundle:^3.1 \
+    php-http/guzzle7-adapter \
+    terminal42/service-annotation-bundle:^1.1 \
+    toflar/psr6-symfony-http-cache-store:^4.0 \
+    twig/twig:^3.11
```

Comment:
```suggestion
    php-http/guzzle7-adapter \
    toflar/psr6-symfony-http-cache-store:^4.0
```

These are required by the `contao/core-bundle` anyway, so I don't think you would need to manually require them?

Reply:
I always require transient dependencies directly. But I can remove them again if you think this is wrong ☺️

Reply:
Oh ok, you mean because you would use Twig anyway and directly configure monolog in the App. Then yes you are right.

However, I would at least remove `terminal42/service-annotation-bundle` as that is deprecated anyway in favor of the PHP attributes :)